src/client/src/appStyles.module.css

```diff
@@ -14,10 +14,12 @@
   height: calc(100vh - 124px);
   justify-content: flex-start;
   position: relative;
+  padding-right: 500px;
 }
 
 .centerViewAzurePage {
   padding: 0;
+  padding-right: 500px;
```

Comment:
Will this be responsive?

Reply:
This is the width of the sidebar. I had to add it here as I changed it's position to fixed.
It keeps the responsiveness of main content :)